test/common/components/RequiredTermsAcceptance.unit.spec.jsx

```diff
@@ -18,6 +18,9 @@ const defaultProps = {
 };
 
 describe('<RequiredTermsAcceptanceView>', () => {
+
+  global.window.dataLayer = [];
+
```

Comment:
I have no idea why this test was failing. I don't think it has anything to do with this PR because it also failed when I switched to master and reran it there locally...so I think another test was initializing `window.dataLayer` and this test would usually run after that.